roles/network_plugin/flannel/handlers/main.yml

```diff
@@ -26,6 +27,11 @@
   service:
     name: docker
     state: restarted
+  when: not is_atomic
+
+- name: Flannel | reload docker (atomic)
+  shell: systemctl stop docker && runc list | awk '!/ID/ {print $1}' | xargs -n 1 -I ID runc kill ID KILL && systemctl start docker
```

Comment:
Agreed that would be cleaner. Set live-restore to false before writing flannel configs and switch it back to true after.